docs/configuration.asciidoc

```diff
@@ -441,6 +441,9 @@ exceeds this configured value. While this is very helpful to find the exact
 place in your code that causes the span, collecting this stack trace does have
 some processing and storage overhead.
 
+NOTE: This configuration was previously known as `ELASTIC_APM_SPAN_FRAMES_MIN_DURATION`, which has been deprecated and will be removed in a future major
```

Comment:
Can you please add the old docs back in? While doing that, please rename this config's anchor to `[[config-span-stack-trace-min-duration]]`, and use `[[config-span-frames-min-duration-ms]]` for the deprecated config.

There's special syntax for deprecation notices which we should probably be using: https://github.com/elastic/docs#additions-and-deprecations

Reply:
Oh I see, thanks for spotting this! I've updated the doc 👍